src/lib/util.js

```diff
@@ -70,7 +70,8 @@ async function untar(tarFile, destinationPath, deleteZipWhenDone = false) {
 
 function execInteractive(cwd, args, env) {
   const {spawn} = require('child_process');
-  const shell = spawn(cwd, args, {
+  console.log(cwd);
```

Comment:
Did you mean to leave this in?

Reply:
Nope. I actually noticed and removed already :)